meeting-notes/2018/2018-08-20--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,131 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 August 20, 2018
+
+- **Lead:** Alan Shaw (@alanshaw)
+- **Notetaker:** @jacobheun
+- **Attendees:**
+  - @achingbrain
+  - @jacobheun
+  - @vmx
+  - @mgoelzer
+  - _attendee names..._
+- **Recording:** https://youtu.be/KZ5mMAdMmo0
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+## Notes
+
+### Week Update
+
+@vmx
+  - Done:
+    - (already last week): Describe your DAG in a text file: https://github.com/vmx/dagbuilder
+    - Hello world libp2p (for GraphSync)
+    - FOSS4G talk preparations
+  - Blocked: nope
+  - Next:
+    - FOSS4G conference + holidays (will be away till 2018-09-09)
+
+@jacobheun
+  - Done:
+    - Released [multistream-select@0.14.3](https://github.com/multiformats/js-multistream-select/commit/64206162b525ba841e77dd4a6caa3a73b020e089) to prevent 'already piped' errors from crashing daemons.
+    - Release of libp2p@v0.23.1, which includes @alanshaws fix for bad multiaddrs throwing errors instead of calling back.
+    - Created [ipfs-api#837](https://github.com/ipfs/js-ipfs-api/pull/837) to make mocking the api requests easier, which should make it easier to create tests without spinning up full nodes.
+    - libp2p factory is ready to merge [js-ipfs#1470](https://github.com/ipfs/js-ipfs/pull/1470)
+    - delegate peer routing test updates are ready to merge https://github.com/libp2p/js-libp2p-delegated-peer-routing/pull/3
+    - Found an inconsistency issue when adding libp2p tests for delegate content routing.  Fixed that, going to finish the libp2p tests for that today.
+  - Blocked:
+    - Go delegate route endpoints https://github.com/ipfs/go-ipfs/pull/4595. We can get the work finished for delegated peer/content routing, but it won't be functional until this is deployed.
```

Comment:
@jacobheun can you open an issue on ipfs/infrastructure to track this ask?

Reply:
Done, https://github.com/ipfs/infrastructure/issues/426, with links off to the blocking go-ipfs issue and js-libp2p core issue.